Sources/NIOPosix/NIOThreadPool.swift

```diff
@@ -292,6 +292,48 @@ extension NIOThreadPool {
     }
 }
 
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension NIOThreadPool {
+    #if swift(>=5.7)
```

Comment:
We no longer support older versions of Swift than 5.7, so we can drop this conditional compilation step.

Reply:
ok, I've removed the older version